turbo/generators/config.ts

```diff
@@ -59,24 +91,30 @@ module.exports = function (plop) {
         name: 'name',
         type: 'input',
         message: 'Url path with dashes. E.g. circular-structure',
+        validate: validateNonEmptyString('path'),
       },
       {
         name: 'title',
         type: 'input',
         message: 'Title for the error. E.g. Circular Structure',
+        validate: validateNonEmptyString('title'),
       },
       {
         name: 'why',
         type: 'input',
         message: 'What caused the error to happen?',
+        validate: validateNonEmptyString('why'),
       },
       {
         name: 'fix',
         type: 'input',
         message: 'What are the possible ways to fix it?',
+        validate: validateNonEmptyString('fix'),
       },
     ],
-    actions: function ({ name }) {
+    actions: function (answers) {
+      const { name } = answers as ErrorResponse
```

Comment:
Why does this need `as ErrorRespose`?

Are the types wrong?

Reply:
`answers` is always just `Record<string, any>` (from inquirer) and it's not a generic. So we're overloading to get more accurate types.